modules/t/sliceAdaptor.t

```diff
@@ -56,6 +55,29 @@ ok($slice->seq_region_length == 62842997);
 debug("slice seq_region length = " . $slice->seq_region_length());
 
 
+#
+# _fetch_by_seq_region_synonym
+#
+
+{
+  my $synonym = "anoth_20";
+  my $start = "1";
+  my $end = "1e6";
+  my $strand = "1";
+  my $target_seq_name = "20";
+  my $target_cs_name = "chromosome";
+  my $synonym_slice = $slice_adaptor->_fetch_by_seq_region_synonym( undef, $synonym, $start, $end, $strand, undef, undef );
+  test_slice("${target_seq_name}:${start}-${end}", $synonym_slice, $target_cs_name, $target_seq_name, $start, $end, $strand);
+}
+
```

Comment:
I would add one more test here to get through the synonym-wildcard code path (maybe make one for `$synonym = "anoth"`

Reply:
Otherwise, this looks good

Reply:
Thanks Brandon, will add the suggested synonym-wildcard test in.